src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+spring.application.name=newsaggregator
+
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=password
+
+spring.security.jwt.secret-key=404E635266556A586E3272357538782F413F4428472B4B6250645367566B5970
```

Comment:
Just FYI: The original token values are generally not uploaded to remote repos. They are managed through some cloud managed config key stores